src/components/Dropdown/__tests__/__snapshots__/Dropdown.test.tsx.snap

```diff
@@ -1606,7 +1606,7 @@ exports[`Dropdown closes options when clicking outside 2`] = `
   transition: filter .5s,box-shadow .5s;
   contain: layout;
   position: relative;
-  z-index: 1;
+  z-index: 7;
```

Comment:
The snapshot changing was worrying, but IIRC this change makes it more correct..